PyDSS/dssObjectBase.py

```diff
@@ -93,10 +94,14 @@ def UpdateValue(self, VarName):
             cachedValue = self.GetValue(VarName, convert=True)
             self._CachedValueStorage[VarName] = cachedValue
         else:
-            value = self.GetValue(VarName, convert=False)
-            if isinstance(cachedValue, ValueByNumber) and VarName in self.VARIABLE_OUTPUTS_COMPLEX:
-                value = complex(value[0], value[1])
-            cachedValue.set_value_from_raw(value)
+            if VarName.lower()=='taps':
```

Comment:
@nadiavp I think I fixed this in a better way in the master branch, so I plan to remove this code. It does mean that you will need to re-test.